packages/liquidator/index.js

```diff
@@ -60,35 +60,45 @@ async function run({
     const voting = new web3.eth.Contract(getAbi("Voting"), getAddress("Voting", networkId));
     const emp = new web3.eth.Contract(getAbi("ExpiringMultiParty"), empAddress);
 
+    // Returns whether the EMP has expired yet
+    const checkIsExpiredPromise = async () => {
```

Comment:
@mrice32 I went ahead and wrapped this in a promise which we can add to the parallel promise below. Thought this was cleaner

Reply:
nit: why not just `checkIsExpired`? IMO, calling it `promise` makes it sound like it's a promise rather than a function.